spec.md

```diff
@@ -752,7 +757,9 @@ A Controller Plugin MUST implement this RPC call if it has `CREATE_DELETE_VOLUME
 This RPC will be called by the CO to provision a new volume on behalf of a user (to be consumed as either a block device or a mounted filesystem).
 
 This operation MUST be idempotent.
-If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities`, `parameters` and `mutable_parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+
+The `parameters` field SHALL contain opaque volume attributes to be specified at creation time. This field SHOULD be used to specify features of the volume that SHOULD NOT change. The `mutable_parameters` field SHALL contain opaque volume attributes attributes that are defined at creation time but MAY also to be changed during the lifetime of the volume via a subsequent `ModifyVolume` RPC. Values specified in `mutable_parameters` SHOULD be applied after the values from `parameters`.
```

Comment:
(1) one sentence per line, please

(2) "applied after" - see earlier comment re: alt wording

Reply:
(2) one sentence per line, do we need to add a new line between the sentences too?

Reply:
> (2) one sentence per line, do we need to add a new line between the sentences too?

https://github.com/container-storage-interface/spec/blob/master/CONTRIBUTING.md#markdown-style